README.md

```diff
@@ -15,12 +15,18 @@ The backend is built with Ruby on Rails while the frontend is built with React.
 
 ## How to run
 
-### Backend 
-``asdf install``
-``bundle install``
-``bin/rails server``
+### Backend
+
+`asdf install`
+`bundle install`
+`bin/rails server`
 
 ### Frontend
-``cd frontend/``
-``npm install``
-``npm start``
+
+`cd frontend/`
+`npm install`
+`npm start`
+
+### Eslint, Prettier and Rubocop 
+
+`./bin/lint`
```

Comment:
What is this line supposed to tell the developer trying to set up your project? What do you want them to know? Where the script is located? How to run the script?